superbench/common/utils/topo_aware.py

```diff
@@ -31,6 +34,32 @@ def search(self, pattern, string, flags=0):
         return self.matched
 
 
+def gen_ibstat_file(ibstat_file):
+    """Generate ibstat file for each node with specified path.
+
+    Args:
+        ibstat_file (str): path of ibstat output.
+    """
```

Comment:
Although MPI_Init() is actually called when we import MPI module, should we check if it's initialized successfully?

Reply:
Thanks, updated.